File: theFinale.client/src/models/Keep.js
export class Keep{
  constructor(data) {
    this.id = data.id
    this.creatorId = data.creatorId
    this.name = data.name
    this.description = data.description
    this.img = data.img
    this.views = data.views
    // virtuals section below
    this.kept = data.kept
    this.creator = data.creator
  }
}

File: theFinale/Models/Keep.cs
namespace theFinale.Models
{
  public class Keep
  {
    public int id { get; set; }
    public string creatorId { get; set; }
    public string name { get; set; }
    public string description { get; set; }
    public string img { get; set; }
    public int views { get; set; }

    // virtuals section below

    public int kept { get; set; }
    public Account creator { get; set; }

  }

  public class vk : Keep
  {
    public int vaultKeepId { get; set; }
  }
}